lang/cs/Org.Apache.REEF.Wake/Org.Apache.REEF.Wake.csproj

```diff
@@ -27,9 +27,17 @@ under the License.
     <FileAlignment>512</FileAlignment>
     <RestorePackages>true</RestorePackages>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..</SolutionDir>
+    <AvroBinaryDirectory>..\packages\AvroBin</AvroBinaryDirectory>
```

Comment:
Turns out this does not work as for some reason the macro expands to nothing when passed to an MSBuild copy task.